File: SpringBootApp/src/main/java/com/example/springbootapp/data/dto/AddressDto.java
package com.example.springbootapp.data.dto;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class AddressDto {
    private String id;
    private String street;
    private String city;
    private String civicNumber;
    private String zipCode;
    private String state;
    private Boolean defaultAddress;
}

File: ClientUser/app/src/main/java/com/example/clientuser/model/dto/AddressDto.kt
package com.example.clientuser.model.dto

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class AddressDto (
    val id: String,
    val street: String,
    val city: String,
    val civicNumber: String,
    val zipCode: String,
    val state: String,
    val defaultAddress: Boolean
)